.github/workflows/release.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         id: goreleaser
         uses: goreleaser/goreleaser-action@286f3b13b1b49da4ac219696163fb8c1c93e1200 # v6.0.0
         with:
-          version: 1.21.x
+          version: 1.22.x
```

Comment:
This is **not** the Go version BTW, but the goreleaser version.

Ref: https://github.com/getsops/sops/issues/1547#issuecomment-2196460328

Reply:
Interesting, this is confusing, why would you use a goreleaser version from Sept 2023 if 1.26.x is the last 1.x? 🤔
I suggest "updating" to 1.26 first, but 2.1 is around the corner...